akd_mysql/src/mysql.rs

```diff
@@ -385,6 +385,28 @@ impl<'a> AsyncMySqlDatabase {
         Ok(())
     }
 
+    /// Drop all the tables
+    pub async fn drop_tables(&self) -> core::result::Result<(), MySqlError> {
```

Comment:
Isn't this just the same as the ```test_cleanup``` method?

Can we drop ```test_cleanup``` then and just make this a public method "drop_tables"? In the unit tests and integration tests we'll want to change the call to ```test_cleanup```